sympy/functions/special/tests/test_gamma_functions.py

```diff
@@ -65,8 +65,10 @@ def test_gamma():
     # Test a bug:
     assert expand_func(gamma(x + Rational(3, 4))) == gamma(x + Rational(3, 4))
 
+    # XXX: Not sure about these tests. I can fix them by defining e.g.
+    # exp_polar.is_integer but I'm not sure if that makes sense.
     assert gamma(3*exp_polar(I*pi)/4).is_nonnegative is False
-    assert gamma(3*exp_polar(I*pi)/4).is_nonpositive is True
+    assert gamma(3*exp_polar(I*pi)/4).is_extended_nonpositive is True
```

Comment:
Does it make sense to define exp_polar.is_integer? If so should it exp_polar(x).is_integer be the same as exp_polar(x)?

Reply:
It is not easy to say what that would mean. Technically, exp_polar is not even complex (except possibly for real arguments).

Reply:
Does it even make sense to use exp_polar as an argument to the gamma function?